### PR TITLE
bpo-33284 Add to a unit test to improve coverage for numbers.py

### PR DESCRIPTION
**Pull request by londinburgh**
Small change to improve unit test coverage. My first merge request for python. 

https://bugs.python.org/issue33284

### CODE REVIEW

**Comment by londinburgh** [comment]
Hi, I have signed the CLA.

**Comment by zware** [comment]
Hi @londinburgh.  While you are all set up on the CLA, the committer (@trevaskis) is not, and @the-knights-who-say-ni requires that all committers and authors on a PR have the CLA signed.

**Comment by londinburgh** [comment]
Ok, now all contributors have signed!

**Comment by londinburgh** [comment]
Have updated this.

File: Lib/test/test_abstract_numbers.py [hunk 1]
```diff
@@ -40,5 +41,92 @@ def test_complex(self):
...
+class TestAbstractNumbers(unittest.TestCase):
+    def testComplex(self):
+
+        class SubComplex(Complex):
+            def __complex__(self):
+                pass
+
+            @property
+            def real(self):
+                return self._imag
```
Comment (BoboTiG) [inline Lib/test/test_abstract_numbers.py:53]: It should be `return self._real`, right?

File: Lib/test/test_abstract_numbers.py [hunk 2]
```diff
@@ -40,5 +41,92 @@ def test_complex(self):
...
+
+            @property
+            def imag(self):
+                return self._real
```
Comment (BoboTiG) [inline Lib/test/test_abstract_numbers.py:57]: And same here with `self._imag`.

**Review by BoboTiG — COMMENTED** [review]
Thanks for the patch @londinburgh. I did not test as I think there are 2 little mistakes :)

Also it may be worth adding a NEWs entry?

**Comment by csabella** [comment]
@londinburgh, please make the changes requested by @BoboTiG.  Thanks!

File: Lib/test/test_abstract_numbers.py [hunk 3]
```diff
@@ -40,5 +41,92 @@ def test_complex(self):
...
+            def __eq__(self, other):
+                return isinstance(other, SubComplex) and self.imag == other.imag and self.real == other.real
+
+            @imag.setter
+            def imag(self, value):
+                self._imag = value
+
+            @real.setter
+            def real(self, value):
+                self._real = value
```
Comment (zware) [inline Lib/test/test_abstract_numbers.py:108]: These setters would be better placed alongside the getters.

**Review by zware — CHANGES_REQUESTED** [review]

**Comment by csabella** [comment]
Since this PR is against an unknown repository, I'm going to close it.  A new pull request can be opened for the issue, but if it's based on this PR, please credit the original author.

**Pull request closed without merging**